package.json

```diff
@@ -22,8 +23,7 @@
         "mathjs": "^9.5.0",
         "minimist": "^1.2.5",
         "oclif": "^2.4.3",
-        "playwright": "1.30.0",
-        "vm2": "3.9.16"
+        "playwright": "1.30.0"
```

Comment:
It should have the same version than `@playwright/test`

Reply:
:+1: